### DISCUSSION TITLE
Some problems running on win10

### DISCUSSION THREAD

**Discussion by redpenguin** [muety/wakapi#315] [Q&A] [answered]
Hi,
I executed wakapi_win_amd64 on win10. It can run, but some functions not working.
For example, clicking on an item inside setting doesn't respond.
The following picture is a screenshot of my execution under two different systems.
![image](https://user-images.githubusercontent.com/8121805/150720866-bfd1a2ba-1f81-4d44-8d1e-ee479d4cb21c.png)
Because my server is under win10, is there a solution?
I tried compiler source code, but to no avail.

**Comment by muety** [comment, marked as answer]
This might actually be the same issue as https://github.com/muety/wakapi/issues/294. Can you please inspect your browsers tools' (F12) network- and console tab and post a screenshot?

**Reply by redpenguin** [reply]
Yes,I fixed it.
I modify registry items.
From ``[HKEY_CLASSES_ROOT\.js]
"Content Type"="text/plain"`` to `[HKEY_CLASSES_ROOT\.js]
"Content Type"="text/javascript"`

OS: Win10 20H2

Thank you.
